pkgs/applications/version-management/redmine/gemset.nix

```diff
@@ -405,12 +395,12 @@
     version = "2.5.8";
   };
   nokogiri = {
-    dependencies = ["mini_portile2" "racc"];
+    dependencies = ["racc"];
     groups = ["default" "test"];
     platforms = [];
     source = {
       remotes = ["https://rubygems.org"];
-      sha256 = "1vrn31385ix5k9b0yalnlzv360isv6dincbcvi8psllnwz4sjxj9";
```

Comment:
Are you seeing this too?
```
error: hash mismatch in fixed-output derivation '/nix/store/1h2cfzdd32rhvnlwkq16z84rc525rri9-nokogiri-1.11.7.gem.drv':
         specified: sha256-etl0Hnov7h/7SksuILAOh5ksnv2Wn1V8o7g/smU7m/w=
            got:    sha256-SXapyeeWUn1R3GwxG5vZOgIz9qeWKg9WmqXHgkYYNu8=

```

Reply:
Just tried to build. Yes, I'm getting this too.

Reply:
Rerunning bundix doesn't help.

Reply:
So why is `bundix` pulling the wrong hash... :thinking: I'll have to get back to this at some point. If you find anything please ping.

Reply:
I think I found the issue... For some reason, bundix doesn't update the version string of nokogiri. It's still 1.11.7.

Reply:
The output of bundix says 1.11.7 is downloaded. So my guess is wrong.
```
path is '/nix/store/mix7sp02x07130wdc6rfyma3l9ndb0dc-nokogiri-1.11.7-x86_64-linux.gem'
1z4v7djv4gxqldy5b7wnzng2r6c71sq20bjb9bxizvigg8g79nbs => nokogiri-1.11.7-x86_64-linux.gem
```

Reply:
Now I found the issue :)

This is what bundix downloads:
```
$ nix-prefetch-url https://rubygems.org/downloads/nokogiri-1.11.7-x86_64-linux.gem
[3.4 MiB DL]
path is '/nix/store/mix7sp02x07130wdc6rfyma3l9ndb0dc-nokogiri-1.11.7-x86_64-linux.gem'
1z4v7djv4gxqldy5b7wnzng2r6c71sq20bjb9bxizvigg8g79nbs
```

This is what Nix downloads:
```
$ nix-prefetch-url https://rubygems.org/downloads/nokogiri-1.11.7.gem
[8.9 MiB DL]
path is '/nix/store/a35hgacdgwh9a52mx6ssf0ijb60axkcn-nokogiri-1.11.7.gem'
1vrn31385ix5k9b0yalnlzv360isv6dincbcvi8psllnwz4sjxj9
```

Maybe this is related to the line in Gemfile.lock I marked before? But this file is also generated by bundix...
```
PLATFORMS
-  ruby
+  x86_64-linux
```

Reply:
I changed according lines manually now. This is not the right place to discuss this.

Reply:
So we're going to run into this issue again next time we update `redmine`, right? Should we talk to upstream `bundix` and see what we need to adjust?

Reply:
Most likely, yes. Let's reach out to the bundix developers.